libr/bin/format/mach0/mach0.c

```diff
@@ -2826,7 +2826,8 @@ const struct symbol_t *MACH0_(get_symbols)(struct MACH0_(obj_t) *bin) {
 }
 
 static int parse_import_ptr(struct MACH0_(obj_t) *bin, struct reloc_t *reloc, int idx) {
-	int i, j, sym, wordsize;
+	int i, j;
+	size_t sym, wordsize;
```

Comment:
Is this right? It seems to me the value of `j` is assigned to `sym` few lines below, so they should be of the same time. Also, `sym` initially has -1 as value, so size_t is probably not the right type.

Reply:
Indeed i will replace his type by ssize_t